app/src/androidTest/java/com/android/sample/ui/activityDetails/CommentItemTest.kt

```diff
@@ -90,7 +96,8 @@ class CommentItemTest {
           comments = listOf(testComment),
           onReplyComment = { _, _ -> },
           onDeleteComment = {},
```

Comment:
Adding a creatorId parameter to CommentItem and related test cases to identify the comment's creator.